libs/jwst-codec/src/doc/store.rs

```diff
@@ -807,7 +807,9 @@ impl DocStore {
                 let _ = mem::replace(&mut items[idx], Node::new_gc(item.id, item.len()));
             } else {
                 let mut item = unsafe { item_ref.get_mut_unchecked() };
-                item.content = Arc::new(Content::Deleted(item.len()));
+                if !matches!(item.content.as_ref(), Content::Type(_)) {
```

Comment:
yjs doesn't have such specific logic for types. We may have some other issue in gc logic

Reply:
fixed in https://github.com/toeverything/OctoBase/pull/533/commits/d28e897ab771267635ed4f8c4756391f82f4b564